src/main/java/com/duckblade/osrs/toa/features/tomb/CursedPhalanxDetector.java

```diff
@@ -62,29 +68,21 @@ private void onMenuOptionClicked(final MenuOptionClicked event)
 			return;
 		}
 
-		ItemContainer itemContainer = client.getItemContainer(InventoryID.EQUIPMENT);
+		final ItemContainer eq = client.getItemContainer(InventoryID.EQUIPMENT);
+		final ItemContainer inv = client.getItemContainer(InventoryID.INVENTORY);
 
-		if (itemContainer != null)
+		if (eq == null || inv == null)
```

Comment:
This if should be removed. It's possible for only one of these to be null, and for the other to be valid. Also, InventoryUtil already does the necessary null checks.

Reply:
Ah ok. I see the null check in one of the overloads, reverted it anyway.